### DISCUSSION TITLE
Do the packages in `/releases` add update repositories?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [TheAssassin/AppImageLauncher#666] [Q&A] [answered]
Do `.RPM` packages - like https://github.com/TheAssassin/AppImageLauncher/releases/download/v2.2.0/appimagelauncher-2.2.0-travis995.0f91801.x86_64.rpm - add an update repository when installed (with `sudo dnf install`)? I ask because TeamViewer's does, as others' do.

I'd rather not need to manually update the package.

**Comment by TheAssassin** [comment, marked as answer]
I will not provide package repositories, way too much effort. The packages are perfectly portable, though, so anybody could make a package repository.

**Reply by TheAssassin** [reply]
Note of transparency: I wrote I wouldn't provide packages, but I meant package repositories. Packages are provided on the release page.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/TheAssassin/AppImageLauncher/discussions/666#discussioncomment-10997262

Thanks. Perhaps there shall someday be a desktop counterpart to [Obtanium](https://github.com/ImranR98/Obtainium) that makes that sufficient. It's not infeasible.

**Reply by TheAssassin** [reply]
I would avoid packaging at all if it didn't have some significant advantages over some home-grown setup system that just puts everything in `/opt`. Making distro packages is the job of distros. That's also the motivation behind making AppImages instead.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/TheAssassin/AppImageLauncher/discussions/666#discussioncomment-10997454

@TheAssassin, have you ever considered whether something like https://build.opensuse.org/ might be easier than GitHub's CI, considering that it's designed entirely to make creating packages for supported distributions easier? I've not utilized either, but I've heard enough anecdotes.

Packaging is a pain: https://xkcd.com/927/

**Reply by TheAssassin** [reply]
I've used OBS for years, both professionally, for hobby use and working on AppImages. It does not contribute anything to the kind of packaging I'm working on (that is, packages that can be installed on any distro). I'm basically making packages that are quite similar to AppImages except they leverage the package management's ecosystem and feature set.
